pkg/backend/display/object_diff.go

```diff
@@ -1268,3 +1274,14 @@ func (p *propertyPrinter) translateYAMLValue(v interface{}) (interface{}, bool)
 		return v, true
 	}
 }
+
+// if string exceeds three lines, truncate and add "..."
+func (p *propertyPrinter) truncatePropertyString(propertyString string) string {
```

Comment:
We should truncate on very long lines too. For example, if the output is a giant JSON blob created for computers, it will be 1 line but a very large number of characters. The exact number is debatable, but I would truncate any line with > 150 characters with a `...`.

Reply:
Great point